types/order_items/index.js

```diff
@@ -0,0 +1,96 @@
+const randomSort = require('../../utils/randomSort');
+const getUniqueId = require('../../utils/getUniqueId');
+
+module.exports = ({ card, tags }) => {
+  const uniqId = getUniqueId(card.question);
+  const questionsHtml = `<ol>${randomSort([...card.answers])
+    .map(
+      el =>
+        `<li draggable="true" ondragend="dragEnd()" data-index="${card.answers.indexOf(
+          el
+        )}" ondragover="dragOver(event)" ondragstart="dragStart(event)">${el}</li>`
+    )
+    .join('')}</ol>`;
+
+  const front = `
+<style>
+#${uniqId}_wrapper.checked {
+  border: 2px solid lightgray;
+  pointer-events: none;
+}
+
+li[draggable].error {
+  border-color: red;
+}
+
+li[draggable] {
+  border: 1px solid lightblue;
+  padding: 5px;
+  user-select: none;
+}
+
+[draggable].over {
+  border: 2px dashed #000;
+}
+
+ol {
+  text-align: left;
+}
+</style>
+<div id="${uniqId}_wrapper">
+  <p>${card.question}</p>
+  <div class="questions-wrapper">${questionsHtml}</div>
+</div>
+<button id="${uniqId}_checkBtn">Check</button>
+<div id="log"></div>
+<script>
+var _el;
+var log = document.querySelector('#log');
+
+function dragOver(e) {
+  if (isBefore(_el, e.target))
+    e.target.parentNode.insertBefore(_el, e.target);
+  else
+    e.target.parentNode.insertBefore(_el, e.target.nextSibling);
+}
```

Comment:
Do you think is it reasonable to replace with?
```
 e.target.parentNode.insertBefore(_el, 
      isBefore(_el, e.target) ? e.target : e.target.nextSibling
    );
}
```

Reply:
again - I'm ready to do that after moving to includes, when js/css will be in separate files (with linting and so on )